plugins/discourse-lazy-videos/lib/lazy-videos/lazy_youtube.rb

```diff
@@ -21,11 +21,13 @@ def to_html
       end
 
       escaped_title = ERB::Util.html_escape(video_title)
+      escaped_start_time = ERB::Util.html_escape(params["t"] || 0)
```

Comment:
This is probably overkill but still feels safer, and always defaulting to 0 make it explicit down the chain without having to deal with presence of the attribute or not.